app/scripts/modules/core/src/pipeline/config/triggers/triggers.html

```diff
@@ -4,50 +4,55 @@
       <div class="col-md-11 col-md-offset-1">
         <div class="checkbox">
           <label>
-            <input type="checkbox" ng-model="pipeline.limitConcurrent">
+            <input type="checkbox" ng-model="pipeline.limitConcurrent" />
             <strong>Disable concurrent pipeline executions (only run one at a time). </strong>
           </label>
         </div>
         <div ng-if="pipeline.limitConcurrent" class="checkbox">
           <label>
-            <input type="checkbox" ng-model="pipeline.keepWaitingPipelines">
+            <input type="checkbox" ng-model="pipeline.keepWaitingPipelines" />
             <strong>Do not automatically cancel pipelines waiting in queue.</strong>
             <help-field key="pipeline.config.parallel.cancel.queue"></help-field>
           </label>
         </div>
       </div>
     </div>
   </page-section>
-  <render-if-feature feature="artifacts">
```

Comment:
Most of this file is prettier changes, but this page section is taken out of the `<render-if-feature>` and instead relies on `checkFeatureFlag`